### PR TITLE
feature:  home page with integrated category & product services bug solved, product form details is next PR

### PR DESCRIPTION
**Pull request by kokobear3327**

### CODE REVIEW

File: src/app/products/products.component.ts [hunk 1]
```diff
@@ -1,7 +1,25 @@
...
+import { ProductService } from './../product.service';
 import { Component } from '@angular/core';
 
 @Component({
   selector: 'app-products',
   templateUrl: './products.component.html'
 })
-export class ProductsComponent {}
+export class ProductsComponent {
+  products$;
```
Comment (ahaly92) [inline src/app/products/products.component.ts]: these should have types. also i could be wrong but aren't the variables with $ signs supposed to be of type observable?

Comment (ahaly92) [reply src/app/products/products.component.ts]: then you run em through angular's async pipe. 

`products$ = categoryService
      .getCategories()
      .valueChanges()` and in the html you would do` {product$ | async }`

File: src/app/admin/product-form/product-form.component.ts [hunk 2]
```diff
@@ -6,17 +8,21 @@ import { Component } from '@angular/core';
...
 })
 export class ProductFormComponent {
-  categories: String[] = [
-    'Bread',
-    'Dairy',
-    'Fruits',
-    'Seasonings',
-    'Vegetables'
-  ];
+  categories$;
```
Comment (ahaly92) [inline src/app/admin/product-form/product-form.component.ts]: this should be of type observable and you should pass through an async pipe

File: src/app/products/products.component.ts (hunk 1 above)
Comment (kokobear3327) [reply src/app/products/products.component.ts]: I will set up types tomorrow, but may need assistance with async method as the error says cannot run async on [object object] in console.  I will arrive tomorrow early and try to tackle again.

**Review by ahaly92 — APPROVED** [review]

**Pull request merged**